### PR TITLE
request to merge final abaum1 code with prospector master

### PR DESCRIPTION
**Pull request by abaum1**

### CODE REVIEW

File: prospector-frontend/src/jslib/maplib.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ let sfmap = L.map('sfmap').setView([37.77, -122.42], 12);
 let url = 'https://api.mapbox.com/styles/v1/mapbox/'+theme+'-v9/tiles/256/{z}/{x}/{y}?access_token={accessToken}';
 let token = 'pk.eyJ1IjoicHNyYyIsImEiOiJjaXFmc2UxanMwM3F6ZnJtMWp3MjBvZHNrIn0._Dmske9er0ounTbBmdRrRQ';
 let attribution ='<a href="http://openstreetmap.org">OpenStreetMap</a> | ' +
-                 '<a href="http://mapbox.com">Mapbox</a>';
+                 '<a href="http://mapbox.com">Mapbox</a> | ' + '<a href="https://github.com/abaum1">Amelia Baum</a>';
```
Comment (bhargavasana) [inline prospector-frontend/src/jslib/maplib.js]: Hi Amelia,
We are currently not attributing individual authors at this location. Also, this change would affect other visualizations. Could you please remove this and commit again?

Comment (bhargavasana) [reply prospector-frontend/src/jslib/maplib.js]: Sorry, I did not realize that you could no longer work on this. We will make some edits on our end. Thanks!

**Comment by abaum1** [comment]
I didn't realize that either! So, just to clarify, you don't need me to
edit and submit another one?

**Comment by bhargavasana** [comment]
Yes, I made a few edits after merging the pull request since they are really minor. You don't need to edit and submit another one. Please note that if you would like to work on this and submit something else in future, 1. either update your fork (a bunch of git commands ) first or 2. delete and re-fork from the TA prospector repo. Thanks!

**Comment by abaum1** [comment]
Great, thank you! I will remember that about updating the fork. Looking
forward to seeing how this finishes up!

Amelia

**Pull request merged**